packages/state-transition/src/block/processAttestationPhase0.ts

```diff
@@ -80,7 +84,14 @@ export function validateAttestation(state: CachedBeaconStateAllForks, attestatio
         `targetEpoch=${data.target.epoch} computedEpoch=${computedEpoch}`
     );
   }
-  if (!(data.slot + MIN_ATTESTATION_INCLUSION_DELAY <= slot && slot <= data.slot + SLOTS_PER_EPOCH)) {
+
+  // post deneb, the attestations are valid till end of next epoch
+  if (
+    !(
+      data.slot + MIN_ATTESTATION_INCLUSION_DELAY <= slot &&
+      (fork >= ForkSeq.deneb || slot <= data.slot + SLOTS_PER_EPOCH)
```

Comment:
Does there need to be a check for the extra epoch worth of slots if deneb here?

Reply:
same reason as previous, attestation is already invalid if target not in current or previous epoch